### PR TITLE
🚜 Merges/Redirect Egardia component pages

### PR DESCRIPTION
**Pull request by klaasnicolaas**
**Description:**
Merges components pages for Egardia and add redirects for other component platform pages.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [ ] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [ ] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/egardia.markdown [hunk 1]
```diff
@@ -100,22 +108,28 @@ There seem to be multiple versions of software running on GATE-02 devices; we ha
...
-      report_server_enabled: true
-      report_server_port: PORT_OF_EGARDIASERVER (optional, defaults to 52010)
-      report_server_codes:
-        arm: XXXXXXXXXXXXXXXX, XXXXXXXXXXXXXXXX
-        disarm: XXXXXXXXXXXXXXXX, XXXXXXXXXXXXXXXX
-        armhome: XXXXXXXXXXXXXXXX
-        triggered: XXXXXXXXXXXXXXXX, XXXXXXXXXXXXXXXX, XXXXXXXXXXXXXXXX
-        ignore: XXXXXXXXXXXXXXXX
-    ```
+
```
Comment (frenck) [inline source/_components/egardia.markdown]: This should not have been changed. It breaks the list...

File: source/_components/egardia.markdown [hunk 2]
```diff
@@ -100,22 +108,28 @@ There seem to be multiple versions of software running on GATE-02 devices; we ha
...
 
-5. Test your setup and enjoy. The component will update if the alarm status changes, including triggers. You can use this to build your own automations and send notifications as you wish. *Note*: previous versions required a separate egardiaserver to be set up. This is no longer necessary and corresponding system services can be removed (using systemctl).
+1. Test your setup and enjoy. The component will update if the alarm status changes, including triggers. You can use this to build your own automations and send notifications as you wish. *Note*: previous versions required a separate egardiaserver to be set up. This is no longer necessary and corresponding system services can be removed (using systemctl).
```
Comment (frenck) [inline source/_components/egardia.markdown]: This was the hint that the list break 😉

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_components/egardia.markdown (hunk 1 above)
Comment (klaasnicolaas) [reply source/_components/egardia.markdown]: I thought to do something well in the context of a markdownlint, I adjusted it.

**Review by frenck — APPROVED** [review]

File: source/_components/egardia.markdown (hunk 1 above)
Comment (frenck) [reply source/_components/egardia.markdown]: 👯

**Pull request merged**